src/api/generate-xlsx/map-application-to-XLSX/map-exporter-business/map-different-trading-address/index.ts

```diff
@@ -0,0 +1,28 @@
+import FIELD_IDS from '../../../../constants/field-ids/insurance/business';
+import { XLSX } from '../../../../content-strings';
+import xlsxRow from '../../helpers/xlsx-row';
+import { ApplicationCompany } from '../../../../types';
+
+const {
+  ALTERNATIVE_TRADING_ADDRESS: { FULL_ADDRESS, FULL_ADDRESS_DOT_NOTATION },
+} = FIELD_IDS;
+
+const { FIELDS } = XLSX;
+
+/**
+ * mapDifferentTradingAddress
+ * Generate an XLSX row if a company has a "different trading address" value.
+ * @param {ApplicationCompany} company: Application company
+ * @returns {Object | undefined} xlsxRow
+ */
+const mapDifferentTradingAddress = (company: ApplicationCompany) => {
+  const { differentTradingAddress } = company;
+
+  const differentTradingAddressValue = differentTradingAddress[FULL_ADDRESS];
```

Comment:
do we need  this line or could we just do `if ( differentTradingAddress[FULL_ADDRESS]) {`

Reply:
It's to be DRY, otherwise we'd have `differentTradingAddress[FULL_ADDRESS]` in two places (line 23 and 24)